doc/docusaurus/docusaurus-examples.cabal

```diff
@@ -0,0 +1,27 @@
+cabal-version: 3.0
+name:          docusaurus-examples
+version:       0.1.0.0
+license:       Apache-2.0
+license-file:  LICENSE
+author:        Yura Lazaryev
+maintainer:    Yuriy.Lazaryev@iohk.io
+category:      Language
+build-type:    Simple
+
+common warnings
+  ghc-options: -Wall
+
+executable example-cip57
+  import:           warnings
+  main-is:          Example/Cip57/Blueprint/Main.hs
+  hs-source-dirs:   static/code
+  default-language: GHC2021
```

Comment:
GHC2021 is supported since 9.2.1.
In CI we are still supporting 8.10.7 so you'll have to wrap this in an `if()` or not use it at all I'm afraid.